Frontend/src/pages/signup/Signup.jsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import "./signup.css"
+import "../CSS/login-signup.css"
```

Comment:
Regarding the login/signup CSS file, I will suggest this
- Rename the "CSS" folder to "auth", this folder will be for everything authentication.
- Move the login file and signup file to it.
- update the path on the routing file.


Please do let me know if you need clarity